src/main/scala/com/databricks/labs/overwatch/utils/Structures.scala

```diff
@@ -35,8 +35,37 @@ case class DatabricksContractPrices(
                                      jobsLightDBUCostUSD: Double = 0.10
                                    )
 
-case class ApiEnv(isLocal: Boolean, workspaceURL: String, rawToken: String, packageVersion: String,successBatchSize:Int=50,errorBatchSize:Int=50,runID:String="",enableUnsafeSSL:Boolean=false,threadPoolSize:Int=4)
-
+case class ApiEnv(
+                   isLocal: Boolean,
+                   workspaceURL: String,
+                   rawToken: String,
+                   packageVersion: String,
+                   successBatchSize: Int = 50,
+                   errorBatchSize: Int = 50,
+                   runID: String = "",
+                   enableUnsafeSSL: Boolean = false,
+                   threadPoolSize: Int = 4,
+                   apiWaitingTime: Long = 300000,
+                   proxyHost: Option[String] = None,
+                   proxyPort: Option[Int] = None,
+                   proxyUserName: Option[String] = None,
+                   proxyPasswordScope: Option[String] = None,
+                   proxyPasswordKey: Option[String] = None
+                 )
+
+
+case class ApiEnvConfig(
+                         successBatchSize: Int = 200,
+                         errorBatchSize: Int = 500,
```

Comment:
I think these should be set == the defaults in `ApiEnv` and the defaults in Param Deserializer. Please unify the defaults

Reply:
done